examples/experimental/test/end_to_end/http_server/large_files/test/authorization_tests.ts

```diff
@@ -1,6 +1,6 @@
-import { generateIdentity, getCanisterId } from 'azle/dfx';
-import { createActor } from 'azle/src/build/experimental/commands/upload_assets/file_uploader/uploader_actor';
-import { expect, it, please, Test } from 'azle/test';
+import { createActor } from 'azle/_internal';
```

Comment:
Is there not a better location for this? Actually, should we try to flatten _internal a bit more? Let's discuss. It would be nice to not have so many paths under _internal.